docs/source/notebooks/pcca.ipynb

```diff
@@ -6,15 +6,16 @@
    "source": [
     "# Coarse-graining with PCCA+\n",
     "\n",
-    "Given a Markov state model describing a reversible Markov chain, i.e., given transition matrix $P\\in\\mathbb{R}^{n\\times n}$ with stationary distribution $\\pi$ it obeys the detailed balance condition\n",
+    "Robust Perron Cluster Analysis (PCCA+)<cite data-cite=\"nbpcca-roblitz2013fuzzy\">(Röblitz, 2013)</cite> is a successor the method PCCA in <cite data-cite=\"nbpcca-deuflhard2000identification\">(Deuflhard, 2000)</cite> which appeared to be not robust enough. It is a method which, given a [Markov state model](../index_msm.rst) with transition matrix $P\\in\\mathbb{R}^{n\\times n}$, can cluster together (sometimes also referred to as coarse-grain) the $n$ _micro_states into $m<n$ _metastable sets_ of states between which also a transition matrix \n",
```

Comment:
"which appeared to be not robust enough" is funny. i think what happened wasthe original pcca (2000) assigned things to macrostates according to eigenvector signs. but this was a problem for eigenvector elements around zero. then the next version (2005) called is called "PCCA+" or "robust PCCA" or "robust PCCA+" and has fuzzy clustering/cluster assignment probabilities main to avoid the problem of eigenvector elements near zero. robust pcca(+) was further developed by Röblitz nee Kube

Reply:
thanks!